patterns/template-single-portfolio.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Title: Portfolio Single Template
+ * Slug: twentytwentyfour/template-single-portfolio
+ * Template Types: page, single
```

Comment:
My PR ended up being mostly the same but I have posts/single here. I don't think we should have page here, maybe we should have all 3? we can revisit that